packages/priority-queue/src/index.js

```diff
@@ -1,4 +1,9 @@
-const requestIdleCallback = window.requestIdleCallback ? window.requestIdleCallback : window.requestAnimationFrame;
+const requestIdleCallback = ( function() {
+	if ( typeof window !== 'undefined' ) {
+		return window.requestIdleCallback ? window.requestIdleCallback : window.requestAnimationFrame;
+	}
+	return setTimeout;
```

Comment:
I'm just assuming falling back to setTimeout is ok here. The API to setTimeout includes an optional second parameter, the timeout, which we are not specifying at the call site, but which defaults to 0.

Reply:
> I'm just assuming falling back to setTimeout is ok here. The API to setTimeout includes an optional second parameter, the timeout, which we are not specifying at the call site, but which defaults to 0.

I think it should be fine. Not super problematic since it's internal-only, but we could wrap `setTimeout` if we wanted to avoid issues of whether the interface is capable of handling a second argument.

```suggestion
	return ( callback ) => setTimeout( callback, 0 );
```

Reply:
> we could wrap `setTimeout` if we wanted to avoid issues of whether the interface is capable of handling a second argument.

I would suggest this be implemented.